encoding/ccf/bench_test.go

```diff
@@ -0,0 +1,215 @@
+/*
+ * Cadence - The resource-oriented smart contract programming language
+ *
+ * Copyright 2022-2023 Dapper Labs, Inc.
```

Comment:
We decided to omit the copyright year in the license text (and instead already added it to the NOTICE file): https://github.com/onflow/cadence/pull/2258

```suggestion
 * Copyright Dapper Labs, Inc.
```

Reply:
Done in commit [82e1812](https://github.com/onflow/cadence/pull/2364/commits/82e1812f11a327cea381fb37e155726ee4a41f3c).